server/modules/record/manager/_recordManager/recordValidationManager.js

```diff
@@ -59,9 +64,22 @@ const persistValidation = async (survey, record, nodesValidation, tx) => {
   await RecordRepository.updateValidation(surveyId, Record.getUuid(record), recordValidationUpdated, tx)
 }
 
+const validateOtherRecordsUniquenessAndPersistValidation = async (survey, record, excludeRecordFromCount, t) => {
```

Comment:
should we call this method validateRecordsUniquenessAndPersistValidation?

Reply:
ok, the shorter the better...